src/components/lyne-tab-amount/styles/lyne-tab-amount.component.scss

```diff
@@ -0,0 +1,10 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
+// and default mode
+@include host-component-properties;
+
+:host {
+  padding-left: 4px;
```

Comment:
Sorry, the fixed number in pixels was just temporary.

It would be likely to have the padding set by the component that uses lyne-tab-amount, but being a child of the shadow-root it cannot be selected by the component css rules.

Reply:
Maybe you can use an override variable here as well?

Reply:
Yes, I believe we already did somthing like that when we also fixed the styling for the dir="rtl". So now we have a --lyne-amount-spacing-override var...

Reply:
I used it, can you check if this what you intended in my next commit?

Reply:
It looks good, thanks